packages/babel-helper-split-export-declaration/src/index.ts

```diff
@@ -6,25 +6,32 @@ import {
   variableDeclaration,
   variableDeclarator,
 } from "@babel/types";
-
-export default function splitExportDeclaration(exportDeclaration) {
+import type * as t from "@babel/types";
+import type { NodePath } from "@babel/traverse";
+
+export default function splitExportDeclaration(
+  exportDeclaration: NodePath<
+    t.ExportDefaultDeclaration | t.ExportNamedDeclaration
+  >,
+) {
   if (!exportDeclaration.isExportDeclaration()) {
     throw new Error("Only export declarations can be split.");
+  } else if (exportDeclaration.isExportAllDeclaration()) {
```

Comment:
Since it doesn't make sense to try to split this, can we throw instead of silently returning?